README.md

```diff
@@ -115,6 +115,10 @@ go-pg is a PostgreSQL client and ORM. It includes many features that traditional
 
 pgx is "closer to the metal" and such abstractions are beyond the scope of the pgx project, which first and foremost, aims to be a performant driver and toolkit.
 
+## API documentation
+
+Full go doc style documentation for the package can be viewed online without installing this package by using the GoDev site here: https://pkg.go.dev/github.com/jackc/pgx/v4
```

Comment:
The repo has evolved since this PR was opened, and this URL should now be `https://pkg.go.dev/github.com/jackc/pgx/v5` to mirror the other link at the top of the README (https://github.com/jackc/pgx/commit/72cc95e4dd18576691da38c46c7e6a101869c10a).